content/docs/specs/om/open_metrics_spec_2_0.md

```diff
@@ -224,8 +228,11 @@ Histogram MetricPoints MUST have one bucket with an +Inf threshold. Buckets MUST
 
 The +Inf bucket counts all requests. If present, the Sum value MUST equal the Sum of all the measured event values. Bucket thresholds within a MetricPoint MUST be unique.
 
-Semantically, Sum, and buckets values are counters so MUST NOT be NaN or negative.
-Negative threshold buckets MAY be used, but then the Histogram MetricPoint MUST NOT contain a sum value as it would no longer be a counter semantically. Bucket thresholds MUST NOT equal NaN. Count and bucket values MUST be integers.
+Semantically, buckets values are counters so MUST NOT be NaN or negative.
+
+The Sum is only a counter semantically as long as there are no negative event values measured by the Histogram MetricPoint. The Sum MUST NOT be NaN.
```

Comment:
> The Sum MUST NOT be NaN

This looks to be new, I am not opposed as I don't see much of a reason for allowing a NaN sum, but is there more behind it than that?

Reply:
The old text had 
```
Semantically, Sum, and buckets values are counters so MUST NOT be NaN or negative.
```
, so not new actually. I'm trying to make the smallest change possible in this PR so we can focus and make sound decisions.

Reply:
Apparently I am just bad at reading. Sorry 🤦